### PR TITLE
f-restaurant-card@v0.28.0 - Design review changes

### PR DESCRIPTION
**Pull request by AlixRichards**
### Changed
- Updated colour variables to use non global values
- Updated component margins and spacing
- Updated SVG alignments and sizes

I've also tidied the formatting a little in places

---

## UI Review Checks
- [X] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested
- [X] Chrome (latest)

Before (searchWeb)

Screenshot 2022-03-24 at 14 20 47

Screenshot 2022-03-24 at 14 23 46

Screenshot 2022-03-24 at 14 24 44

After (searchWeb - locally)
Screenshot 2022-03-24 at 14 23 19
Screenshot 2022-03-24 at 14 24 13
Screenshot 2022-03-24 at 14 25 19

### CODE REVIEW

File: packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue [hunk 1]
```diff
@@ -63,19 +72,13 @@
...
-
-                <!-- 'New' label -->
-                <restaurant-tag
-                    v-if="newTagText"
-                    :is-large="true"
-                    :is-uppercase="true"
-                    :text="newTagText"
-                    data-test-id="restaurant-new-badge"
-                    color-scheme="positive" />
+                    :class="[$style['c-restaurantCard-premier-iconWrapper']]">
```
Comment (ashleynolan) [inline packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue]: If this is the only iconWrapper currently, then I'd probably just use `c-restaurantCard-iconWrapper` as a name?

Comment (ashleynolan) [reply packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue]: Or if `c-restaurantCard-premier` isn't used anymore, you could just move that class onto this element and delete it off the legend-icon?

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue [hunk 2]
```diff
@@ -67,16 +67,17 @@ export default {
...
     color: $color;
+
     path {
         fill: $color;
     }
 }
 
 .c-restaurantCard-iconText {
-    @include font-size();
+    @include font-size($font-body-s-paragraph);
```
Comment (ashleynolan) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue]:
```suggestion
    @include font-size(body-s);
```

Should give the same result.

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantCuisines.vue [hunk 3]
```diff
@@ -43,7 +42,7 @@ export default {
     padding-right: spacing();
     margin: 0 spacing(a) 0 0;
     white-space: nowrap;
-    font-size: $font-body-s-paragraph * 1px;
+    @include font-size($font-body-s-paragraph);
```
Comment (ashleynolan) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantCuisines.vue]:
```suggestion
    @include font-size(body-s);
```

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue [hunk 4]
```diff
@@ -164,20 +165,26 @@ export default {
 
 <style lang="scss" module>
 .c-restaurantCard-rating {
-    @include font-size(body-s);
+    @include font-size($font-body-s-paragraph);
```
Comment (ashleynolan) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue]:
```suggestion
    @include font-size(body-s);
```

File: packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue (hunk 1 above)
Comment (AlixRichards) [reply packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue]: It's not the only iconWrapper but I'm wondering If I can change both of them to remove the extra class like you suggest 👍

Comment (AlixRichards) [reply packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue]: I couldn't change the one in ratings but I have updated this, thank you ✨

File: packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue [hunk 5]
```diff
@@ -63,19 +72,12 @@
                 </component>
 
                 <!-- Premier Icon -->
-                <legend-icon
+                <span
```
Comment (jamieomaguire) [inline packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue:75]: for my own understanding - what was the reason for adding a wrapping tag?

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue [hunk 6]
```diff
@@ -171,13 +172,19 @@ export default {
...
+}
+
+.c-restaurantCard-rating-star {
+    width: 100%;
+    height: 100%;
+    display: block;
 
     & path {
-        fill: $color-orange-30;
+        fill: $color-dark-content-brand;
```
Comment (dawidchar) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue]: I've remember being told to not use the dark theme colours 🤔

**Review by dawidchar — DISMISSED** [review]

**Review by dawidchar — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**